twitter/tests/test_tweepy_scraper.py

```diff
@@ -34,4 +35,36 @@ def test_add_user(self):
 
         self.assertEqual(test_user, TwitterUser.query.filter_by(user_name='@TestUser').first())
 
+    def test_parse_status(self):
+        self.maxDiff = None
+        status = Mock(
+            user_name = '@fake_user_name',
+            user_location = 'fake_location',
+            coordinates = 'fake_coordinates',
+            text = 'fake_test',
+            # symbols_in_text = 'fake_symbols_in_fake_text',
+            # hashtags = 'fake_hashtags',
+            created_at = 'fake_created_date',
+            #lead_score = 3,
```

Comment:
When assigning values to attributes on an object like this, or when using keyword values in a function, the standard style is not to include spaces on either side of the equals sign. https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements